### DISCUSSION TITLE
Pass Data through live method ($get) to View in filament 3

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#11512] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

I need to pass template id to the view using $get method , here is the code snippet but it shows that this method only accept query builder 
so can you help with this guys ASAP ? 
 Fieldset::make('previewer')
                                            ->schema([
                                                View::make('package-name::Templates.default')
                                                    ->viewData(['template' => \ZID\PackageName\Models\Template::where('id',$get('template_id'))->first()])
                                                    ->live()
                                                    ->columns(1),
                                            ])
                                            ->live()

**Comment by ArtMin96** [comment]
Why `live()`? Where's your `->view('my-view')`?

**Reply by Hegabovic** [reply]
i need to take the template id from the same form (drop down list)

**Reply by leandrocfe** [reply, marked as answer]
Not sure what you are trying to do, but you can get the template id inside the view
```html
<div>
    {{ $this->data['template_id'] }}
</div>
```
